src/routes/index.ts

```diff
@@ -37,9 +39,17 @@ const getContacts = async () => {
 
 router.get('/contacts', async (req, res) => {
 
-    const contact = await getContacts();
-    res.json(contact);
-    console.log('got contacts');
-    res.end();
+    try
+    {
+        const contact = await ApiCall(getContacts,"string");//await getContacts();
```

Comment:
This will break the endpoint, can the refresh token be put here instead of "string"?